FormCountrySelectMenu.php

```diff
@@ -16,10 +16,17 @@ public function addAttributes($arrAttributes)
         parent::addAttributes($arrAttributes);
         $arrOptions = array(array('label' => ($this->placeholder == '' ? '-' : $this->placeholder), 'value' => ''));
         $arrCountries = $this->getCountries();
+        
+        // allow insert tags to be set as default value, e.g. {{user::country}}
+        // see https://github.com/terminal42/contao-countryselect/issues/6
+        if (strpos($this->varValue, '{{') !== false) {
+            $this->varValue = \Contao\Controller::replaceInsertTags($this->value);
```

Comment:
Is it intentional to use `$this->value` and not `$this->varValue` for the method argument?

Reply:
actually yes! `$this->value` is implemented in `class Widget`.

Reply:
I know it is, but why read from that field but then write to another one?

Reply:
ok that seems reasonable. Should I use varValue or value only?

Reply:
I think we should use `$this->varValue` within the class itself. `$this->value` is only for "external" access.